headers/modsecurity/rules_set_properties.h

```diff
@@ -218,14 +225,27 @@ class RulesSetProperties {
 
     /**
      *
-     *
+     * The ConfigBoolean enumerator the states for configuration boolean values.
+     * The default value is PropertyNotSetConfigBoolean.
      */
     enum ConfigBoolean {
         TrueConfigBoolean,
         FalseConfigBoolean,
         PropertyNotSetConfigBoolean
     };
 
+    /**
+     *
+     * The ConfigXMLParseXmlIntoArgs enumerator defines the states for the configuration
+     * XMLParseXmlIntoArgs values.
+     * The default value is PropertyNotSetConfigXMLParseXmlIntoArgs.
+     */
+    enum ConfigXMLParseXmlIntoArgs {
+        TrueConfigXMLParseXmlIntoArgs,
+        FalseConfigXMLParseXmlIntoArgs,
+        OnlyArgsConfigXMLParseXmlIntoArgs,
+        PropertyNotSetConfigXMLParseXmlIntoArgs
```

Comment:
```suggestion
     * The ConfigXmlParseXmlIntoArgs enumerator defines the states for the configuration
     * XmlParseXmlIntoArgs values.
     * The default value is PropertyNotSetConfigXmlParseXmlIntoArgs.
     */
    enum ConfigXmlParseXmlIntoArgs {
        TrueConfigXmlParseXmlIntoArgs,
        FalseConfigXmlParseXmlIntoArgs,
        OnlyArgsConfigXmlParseXmlIntoArgs,
        PropertyNotSetConfigXmlParseXmlIntoArgs
```

Reply:
I can change this, because this `enum` type is new, but in this case we will deviate from the above-mentioned syntax (`m_SecXML...`). What do you think?

Reply:
Personally, I'd use the "new" spelling. I don't think it matters here.